components/board/server.go

```diff
@@ -215,12 +215,6 @@ func (s *serviceServer) StreamTicks(
 		return err
```

Comment:
Not to myself: the reason we don't need to remove the callbacks explicitly is that they get removed when `server.Context()`, two lines up from here, is canceled.